.snippets/code/builders/interact/ethereum-api/precompiles/erc20/erc20.sol

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: GPL-3.0-only
```

Comment:
Same comment here as on the images. This should be deleted, but I know this is also being removed in the Native Token PR